### PR TITLE
added DCDC status CAN message when DCDC state changes

### PR DESCRIPTION
**Pull request by HappyRedMapleLeaf**
Literally added one line of code. Calls sendCAN_PDU_DCDC_Status() when DCDC switches on -> off or off -> on. Accidentally added some newlines as well.

### CODE REVIEW

**Review by addison-ch — APPROVED** [review]

File: pdu/Src/LTC4110.c [hunk 1]
```diff
@@ -26,18 +27,23 @@ void powerTask(void *pvParameters)
...
+
+            sendCAN_PDU_DCDC_Status();
         }
+        
         watchdogTaskCheckIn(5);
+
         vTaskDelayUntil(&xLastWakeTime, POWER_TASK_INTERVAL_MS);
     }
-}
+}
```
Comment (DanBishara) [inline pdu/Src/LTC4110.c:49]: Red circle means add a line at the end of the file

**Review by DanBishara — APPROVED** [review]

File: pdu/Src/LTC4110.c [hunk 2]
```diff
@@ -16,6 +16,7 @@ volatile bool DC_DC_state = false;
 
 void powerTask(void *pvParameters)
 {
+
```
Comment (aryanGajelli) [inline pdu/Src/LTC4110.c:19]: nit: remove newline

File: pdu/Src/LTC4110.c [hunk 3]
```diff
@@ -26,18 +27,23 @@ void powerTask(void *pvParameters)
     vTaskDelay(100);
 
     TickType_t xLastWakeTime = xTaskGetTickCount();
+
```
Comment (aryanGajelli) [inline pdu/Src/LTC4110.c:30]: nit: remove newline

File: pdu/Src/LTC4110.c [hunk 4]
```diff
@@ -26,18 +27,23 @@ void powerTask(void *pvParameters)
...
                 DEBUG_PRINT("switched to DC to DC\n");
             }
+            
             DC_DC_state = newDCDCState;
+
+            sendCAN_PDU_DCDC_Status();
         }
+        
         watchdogTaskCheckIn(5);
+
```
Comment (aryanGajelli) [inline pdu/Src/LTC4110.c:46]: nit: remove newline

**Review by aryanGajelli — COMMENTED** [review]
Remove the newlines to try and keep as much of the orginal formatting. Once that is done it LGTM for approval

**Review by aryanGajelli — CHANGES_REQUESTED** [review]
Change target to merge into develop

**Review by aryanGajelli — APPROVED** [review]

**Pull request merged**